src/infrastructure/entitys/alert.ts

```diff
@@ -0,0 +1,45 @@
+import { Entity, Column, PrimaryColumn, PrimaryGeneratedColumn, CreateDateColumn, Generated } from 'typeorm';
+import { MPPAlert, Severity } from '../../core/Alert';
+
+@Entity()
+export class Alert {
```

Comment:
I want you to change the name of the class to `SQLAlert` or `TypeormAlert` so there will be a difference in the names between the entity that is connected to the sql repository.